lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -51,6 +51,14 @@ __PACKAGE__->add_columns(
         false_id      => ['0', '-1'],
         default_value => '0',
     },
+    last_login_version => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    feature_informed => {
```

Comment:
so with every new feature you would have to write a migration to reset? that's not going to work - you will need to store *exactly* what feature she was informed about